rosetta/Dockerfile.gemma

```diff
@@ -93,7 +55,7 @@ optax
 protobuf
 tfds-nightly
 tensorflow
-tensorflow-text @ file://$(ls /opt/tensorflow_text*.whl)
```

Comment:
Do we still need to explicitly list `tensorflow-text` as a dependency now the wheels for it are published? (not a big deal)